server/lib/queries.js

```diff
@@ -220,6 +220,7 @@ const getTopBackers = (since, until, tags, limit) => {
     LEFT JOIN "Collectives" collective ON collective.id = t."CollectiveId"
     WHERE
       t.type='CREDIT'
+      AND t."deletedAt" IS NULL
```

Comment:
Had to do this small change to make the tests less sensitive in https://github.com/opencollective/opencollective-api/pull/8027/files#diff-ed0b486411042fde03fcb5a88b2fe2159ae7cf484b0d755f4a88c0a666cafaa2. It seemed desirable to ignore deleted transactions anyway.